modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -75,50 +75,65 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 $ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
 ----
 
-. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step.
+. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step. Following is the sample command for worker:
 +
-For example:
+[NOTE]
+====
+You can create MachineConfig objects for both master and worker machine types.
+====
 +
 [source,terminal]
 ----
-$ cat > /tmp/40-worker-custom-journald.yaml <<EOF
+$ cat << EOF > ./99-worker-journald-configuration.yaml <1>
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker
-  name: 50-corp-journald
+    machineconfiguration.openshift.io/role: worker <2>
+  name: 99-worker-journald-configuration <3>
 spec:
   config:
     ignition:
-      version: 3.1.0
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
     storage:
       files:
       - contents:
-          source: data:text/plain;charset=utf-8;base64,${jrnl_cnf}
-        mode: 0644 <1>
-        overwrite: true
-        path: /etc/systemd/journald.conf <2>
+          source: data:text/plain;charset=utf-8;base64,${jrnl_cnf} <4>
+          verification: {}
+        filesystem: root
+        mode: 0644 <5>
+        path: /etc/systemd/journald.conf
```

Comment:
I have no idea why the journald.conf wasn't updated on workers.  I suggest using /etc/systemd/journald.conf.d/custom.conf. that had been tested.  refer to https://www.freedesktop.org/software/systemd/man/journald.conf.html

Reply:
@anpingli Done!